<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>createjob</class>
 <widget class="QMainWindow" name="createjob">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QMainWindow{
background-color: rgb(192, 221, 190);
}
QPushButton#pushButton{
background-color: black;
color: white;
font: bold 15px;
border-radius: 10px;
transition: 0.2s;
}
QPushButton#pushButton:hover {
background-color: rgb(80,80,80);
font: bold 17px;
}
QPushButton#pushButton:pressed {
background-color: rgb(50,50,50);
}
QPushButton#pushButtone:focus{
box-shadow: 0 0 5px rgba(0,0,255,0.5);
font: bold 15px;
Width: 150px;
}

QPushButton#pushButton_2{
background-color: black;
color: white;
font: bold 15px;
border-radius: 10px;
transition: 0.2s;
}
QPushButton#pushButton_2:hover {
background-color: rgb(80,80,80);
font: bold 17px;
}
QPushButton#pushButton_2:pressed {
background-color: rgb(50,50,50);
}
QPushButton#pushButtone_2:focus{
box-shadow: 0 0 5px rgba(0,0,255,0.5);
font: bold 15px;
Width: 150px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>918</width>
      <height>330</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/pishvazpic.JPG);</string>
    </property>
    <property name="locale">
     <locale language="English" country="TuvaluCountry"/>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>115</y>
      <width>841</width>
      <height>341</height>
     </rect>
    </property>
    <property name="sizePolicy">
     <sizepolicy hsizetype="MinimumExpanding" vsizetype="Ignored">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
      <underline>false</underline>
      <stylestrategy>PreferDefault</stylestrategy>
      <kerning>false</kerning>
     </font>
    </property>
    <property name="focusPolicy">
     <enum>Qt::WheelFocus</enum>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::PreventContextMenu</enum>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="accessibleDescription">
     <string notr="true"/>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(54, 66, 51);
</string>
    </property>
    <property name="locale">
     <locale language="Cornish" country="UnitedKingdom"/>
    </property>
    <property name="title">
     <string notr="true">Enter the information and click to create the desired job.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignBottom|Qt::AlignHCenter</set>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="lineEdit_salary">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>75</y>
       <width>291</width>
       <height>45</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>45</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1111111</width>
       <height>50</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>40</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Enter your password</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: bold 15px;
background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 1px solid #0640ff;
padding: 5px;
margin: 5px;
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>Enter the salary for job</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_password_2">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>45</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(54, 66, 51);
</string>
     </property>
     <property name="text">
      <string>Salary :</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_workplase">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>160</y>
       <width>291</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <underline>false</underline>
       <strikeout>false</strikeout>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: bold 15px;
background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 1px solid #0640ff;
padding: 5px;

margin: 5px;
</string>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <item>
      <property name="text">
       <string>Full-time</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Part-time</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Contract</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Temporary</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Other</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Volunteer</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Internship</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_password">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>130</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(54, 66, 51);
</string>
     </property>
     <property name="text">
      <string>Workplace type :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_job_title">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>75</y>
       <width>291</width>
       <height>45</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>45</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1111111</width>
       <height>50</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>40</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Enter your password</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: bold 15px;
background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 1px solid #0640ff;
padding: 5px;
margin: 5px;
</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>Add the title you are hiring for</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_job_type">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>291</width>
       <height>45</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="mouseTracking">
      <bool>false</bool>
     </property>
     <property name="tabletTracking">
      <bool>false</bool>
     </property>
     <property name="focusPolicy">
      <enum>Qt::ClickFocus</enum>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="acceptDrops">
      <bool>false</bool>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">font: bold 15px;
background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 1px solid #0640ff;
padding: 5px;
margin: 5px;

</string>
     </property>
     <property name="locale">
      <locale language="Danish" country="Greenland"/>
     </property>
     <property name="currentIndex">
      <number>-1</number>
     </property>
     <property name="insertPolicy">
      <enum>QComboBox::InsertAlphabetically</enum>
     </property>
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContentsOnFirstShow</enum>
     </property>
     <item>
      <property name="text">
       <string>On-site</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Remote</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Hybrid</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label_password_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>45</y>
       <width>131</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(54, 66, 51);
</string>
     </property>
     <property name="text">
      <string>job title :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_password_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>130</y>
       <width>171</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(54, 66, 51);
</string>
     </property>
     <property name="text">
      <string>job type :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_username_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>225</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(54, 66, 51);</string>
     </property>
     <property name="text">
      <string>Address :</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="textEdit_address">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>250</y>
       <width>291</width>
       <height>81</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <kerning>false</kerning>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: bold 15px;
border-radius: 8px;
border: 1px solid #0640ff;
padding: 5px;

margin: 5px;

</string>
     </property>
     <property name="placeholderText">
      <string>Please enter your address</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_skills">
     <property name="geometry">
      <rect>
       <x>470</x>
       <y>250</y>
       <width>291</width>
       <height>45</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>200</width>
       <height>45</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1111111</width>
       <height>50</height>
      </size>
     </property>
     <property name="sizeIncrement">
      <size>
       <width>50</width>
       <height>50</height>
      </size>
     </property>
     <property name="baseSize">
      <size>
       <width>40</width>
       <height>50</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>-1</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="toolTip">
      <string>Enter your password</string>
     </property>
     <property name="styleSheet">
      <string notr="true">font: bold 15px;
background-color: rgb(255, 255, 255);
border-radius: 10px;
border: 1px solid #0640ff;
padding: 5px;
margin: 5px;
</string>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Normal</enum>
     </property>
     <property name="placeholderText">
      <string>Enter the required skills</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_password_5">
     <property name="geometry">
      <rect>
       <x>480</x>
       <y>220</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Nirmala UI</family>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(54, 66, 51);
</string>
     </property>
     <property name="text">
      <string>Skills :</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>208</width>
      <height>59</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Magneto</family>
      <pointsize>26</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(52, 123, 110);
background-color: rgb(69,92,80);
color: white;
padding: 5px;
border-radius: 10px;
border-top-left-radius: 40px;
border-bottom-right-radius: 40px;</string>
    </property>
    <property name="text">
     <string>PROJOB</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>460</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-radius: 10px;
border-top-left-radius: 40px;
border-bottom-right-radius: 40px;</string>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>460</y>
      <width>150</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nirmala UI</family>
      <pointsize>-1</pointsize>
      <weight>75</weight>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
border-radius: 10px;
border-top-left-radius: 40px;
border-bottom-right-radius: 40px;</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>918</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
